.ado/jobs/desktop.yml

```
@@ -85,7 +85,7 @@ jobs:
           $newValue = '(FullyQualifiedName!~RNTesterIntegrationTests::)&' + "$(Desktop.IntegrationTests.Filter)"
           Write-Host "##vso[task.setvariable variable=Desktop.IntegrationTests.Filter]$newValue"
         displayName: Update Desktop.IntegrationTests.Filter
-        condition: and(succeeded(), ne(variables['Desktop.IntegrationTests.SkipRNTester'], 'true'))
+        condition: and(succeeded(), eq(variables['Desktop.IntegrationTests.SkipRNTester'], 'true'))
```

I've read this a few times and got confused again in the multiple levels of skip/negate/filter :) 
Would it be okay to update the displayname to be a bit more explicit what is happening?
```suggestion
        displayName: Update Desktop.IntegrationTests.Filter to exclude RNTester Integration tests"
```

---

You got it.